### PR TITLE
Fix nbsp helper

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

**Comment by bgeuken** [comment]
@ChrisBr please review

File: src/api/test/unit/webui/webui_helper_test.rb [hunk 1]
```diff
@@ -55,4 +55,14 @@ def test_escape_project_list_escaped_forbidden_chars
     output = "['&lt;p&gt;home:Iggy&lt;/p&gt;', '&lt;p&gt;This is a paragraph&lt;/p&gt;']"
     assert escape_nested_list(input), output
   end
+
+  def test_format_projectname
```
Comment (ChrisBr) [inline src/api/test/unit/webui/webui_helper_test.rb]: Why is it moved from 
``
 src/api/test/unit/webui_helper_test.rb 
`` 
to 
``
 src/api/test/unit/webui_helper_test.rb 
``
?

Comment (ChrisBr) [inline src/api/test/unit/webui/webui_helper_test.rb]: Ah, got it! Agreed, makes more sense in webui subfolder!

**Comment by ChrisBr** [comment]
LGTM!

**Pull request merged**